### PR TITLE
#384 Removes backtick from db name to get --if-not-exists option working

### PR DESCRIPTION
**Pull request by ghost**
Also adds missing argument to `sprintf()`

### CODE REVIEW

**Comment by xabbuh** [comment]
@ggirtsou Looks like you need to rebase here.

File: Command/CreateDatabaseDoctrineCommand.php [hunk 1]
```diff
@@ -79,10 +79,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $error = false;
         try {
-            $shouldNotCreateDatabase = $ifNotExists && in_array($name, $tmpConnection->getSchemaManager()->listDatabases());
+            $cleanName = str_replace('`', '', $name);
```
Comment (deeky666) [inline Command/CreateDatabaseDoctrineCommand.php]: TBH I would rather consider not explicitly quoting the database identifier at all. IMO this is DBAL/user responsibility anyways. Some platforms even REQUIRE unquoted database identifiers like SQL Anywhere.

Comment (deeky666) [inline Command/CreateDatabaseDoctrineCommand.php]: Also quoting the identifier [here](https://github.com/ggirtsou/DoctrineBundle/blob/master/Command/CreateDatabaseDoctrineCommand.php#L77) does not always enclose the database name in backticks. This is only working on MySQL but not for other platforms!

**Comment by guilhermeblanco** [comment]
Closing as this is a duty of DBAL to do, not the bundle

**Pull request closed without merging**